hid.cpp

```diff
@@ -338,7 +338,7 @@ void USBHIDParser::parse()
 			p += *p + 3;
 			continue;
 		}
-		uint32_t val;
+		uint32_t val = 0;
```

Comment:
That's the wrong fix. Yes, I guess the compiler optimized the double val =0 away, but it's not nice to read and lazy.

There should be a default case instead.